builtin/logical/nomad/backend.go

```diff
@@ -0,0 +1,60 @@
+package nomad
+
+import (
+	"github.com/hashicorp/nomad/api"
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func Factory(conf *logical.BackendConfig) (logical.Backend, error) {
+	b := Backend()
+	if err := b.Setup(conf); err != nil {
+		return nil, err
+	}
+	return b, nil
+}
+
+func Backend() *backend {
+	var b backend
+	b.Backend = &framework.Backend{
```

Comment:
We need to add SealWrapStorage to the config path.  See https://github.com/hashicorp/vault/blob/8004f052da4c68f7c3b926e60c51fc8064e83617/builtin/logical/consul/backend.go#L19-L23.

Reply:
Yes, I discussed with Vishal a bit ago. I'll send a PR

Reply:
You can just add it to this PR. It is all included in OSS.

Reply:
Fixed in 884e25035